controllers/awsmachine_security_groups.go

```diff
@@ -31,7 +31,7 @@ const (
 	// the AdditionalSecurityGroups in the Machine Provider Config.
 	// See https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
 	// for annotation formatting rules.
-	SecurityGroupsLastAppliedAnnotation = "sigs.k8s.io/cluster-api-provider-aws/v2-last-applied-security-groups"
+	SecurityGroupsLastAppliedAnnotation = "sigs.k8s.io/cluster-api-provider-aws-v2-last-applied-security-groups"
```

Comment:
Ah damn. I think adding the `/v2` was just a mistake when search/replacing to adding v2.

I think we have to rollback to the annotation we had before my change

Reply:
Thx for catching that!

Reply:
thnx for the clarification @sbueringer ! pushed changes to revert annotations